api/controllers/height_controller.py

```
@@ -0,0 +1,284 @@
+        """
+        query = "CALL getLatestHeight();"
+
+        try:
+            cursor = self.execute_query(query)
+            height = 0
+
+            for id, start_time, start_height, end_time, end_height in cursor:
+                if end_height == None:
+                    height = start_height
+                else:
+                    height = end_height
```

Ruff

Use ternary operator `height = start_height if end_height == None else end_height` instead of `if`-`else`-block

Replace `if`-`else`-block with `height = start_height if end_height == None else end_height`

(SIM108)